### PR TITLE
Portfolio: finish mobile version

### PR DESCRIPTION
**Pull request by Yidnekachew-cmd**
### In this branch:
- I have added the works section.
- I have added the about myself section.
- And also I have added the images form the Figma design.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,5 +37,165 @@ <h1 class="para-title">I'm Yidnekachew Glad to see you!</h1>
...
+            <p class="card-para">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="card-lang">
+              <li class="lang">html</li>
+              <li class="lang">css</li>
+              <li class="lang">javaScript</li>
+            </ul>
+            <button class="card-btn">See Project</button>
```
Comment (frank1738) [inline index.html:85]: - [ ] Kindly work on the padding of the buttons in all your cards to match the original design. There is no good spacing between the text and the border.

image

### Checkout the original design

image

File: index.html [hunk 2]
```diff
@@ -37,5 +37,165 @@ <h1 class="para-title">I'm Yidnekachew Glad to see you!</h1>
...
+          <a class="about-contact" href="#">LET'S CONNECT</a>
+          <ul class="about-icons">
+            <li><img src="./images/Github.svg" alt="icon" /></li>
+            <li><img src="./images/Linkedin icon.svg" alt="icon" /></li>
+            <li><img src="./images/Peace.svg" alt="icon" /></li>
+            <li><img src="./images/twitter.svg" alt="icon" /></li>
+            <li><img src="./images/M.svg" alt="icon" /></li>
+          </ul>
+          <button class="about-btn">Get my resume</button>
+        </div>
```
Comment (frank1738) [inline index.html:155]: - [ ] In your about me section, Kindly adjust the  `get my resume` button width to match the original design

image

### Checkout the original design

image

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @Yidnekachew-cmd , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Github workflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -37,5 +37,165 @@ <h1 class="para-title">I'm Yidnekachew Glad to see you!</h1>
...
+              </li>
+              <li class="langs">
+                <img src="./images/Ellipse 2.svg" alt="logo" />
+                <h4 class="langs-text">HTML</h4>
+              </li>
+              <li class="langs">
+                <img src="./images/Ellipse 3.svg" alt="logo" />
+                <h4 class="langs-text">CSS</h4>
+              </li>
+            </ul>
```
Comment (AbayomiOlaoye) [inline index.html:176]: - [ ] [OPTIONAL]! I think changing some units e.g. `px` , `rem` , etc here will improve the design of your page, giving more quality user experience.
- [ ] [OPTIONAL] The height on your `lang` is presently more than twice the figma design
- [ ] [OPTIONAL] I suggest changing your `langs-text` to match figm design
- [ ] [OPTIONAL] I also believe that using the correct `color` value for your border will improve the quality of your design. 
- Please see what I am referring to below:
**Your Design**: 
image
**Figma design**: 
image

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @Yidnekachew-cmd :hibiscus: , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so_close


 ## Highlights
- Good commit messages :heavy_check_mark: 
- Fair implementation of feautures :heavy_check_mark: 
- Descriptive PR message :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] Your work can be more professionally appealing if you can remove all commented descriptions from your `ReadMe` file, and probably include all the sections you deleted since you still have then in your `Table of Contents`. A plus would be to ensure that you personalize the contents :+1:  You are almost there, champ :smile:  :muscle: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**